dht/sharder.hh

```diff
@@ -28,28 +28,58 @@
 
 namespace dht {
 
+
+// Utilities for sharding ring partition_range:s
+
+// A ring_position range's data is divided into sub-ranges, where each sub-range's data
+// is owned by a single range. Note that multiple non-overlapping sub-ranges may map to a
+// single range, and some shards may not receive any sub-range.
+//
+// This module provides utilities for determining the sub-ranges to shard mapping. The utilities
+// generate optimal mappings: you will each range that you get is the largest possible, so you
```

Comment:
'you will each range'? Missing a verb, I think. Probably 'you will' should just be removed.

Reply:
yes